share/instruments/instruments.xml

```diff
@@ -64,6 +64,22 @@
                   <genre>jazz</genre>
                   <genre>orchestra</genre>
             </Instrument>
+                  <Instrument id="db-piccolo">
+                        <longName>D♭ Piccolo</longName>
+                        <shortName>D♭ Picc.</shortName>
+                        <description>Piccolo Flute in Db</description>
+                        <musicXMLid>wind.flutes.flute.piccolo.dflat</musicXMLid>
```

Comment:
this xml id doesn't exist. https://www.musicxml.com/for-developers/standard-sounds/